### PR TITLE
Ss add users

### PR DESCRIPTION
**Pull request by sschipke**
# Description

I finished the sign up form and login form. The application now allows users to create their accounts (and the corresponding profiles) as well as log in. 

### Important!!
Right now it is just console logging the response from the server, we'll need to figure out what to do with it.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

I need to add testing for this section

- [ ] Unit Testing
- [x] User Testing

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/NewUserForm.vue [hunk 1]
```diff
@@ -1,26 +1,72 @@
...
         <b-field label="Email">
             <b-input class='input-field' type="email"
+                required
+                placeholder="Please enter your email"
+                v-model="account.email"
                 value=""
                 maxlength="40">
             </b-input>
         </b-field>
+        <b-field label="Proficiency" for="proficiency">
```
Comment (nancylee713) [inline src/components/NewUserForm.vue:16]: There are two down arrows for the proficiency section; i've attached a screenshot below
Screen Shot 2020-01-27 at 3 46 01 PM

**Review by nancylee713 — APPROVED** [review]

File: src/components/NewUserForm.vue (hunk 1 above)
Comment (sschipke) [reply src/components/NewUserForm.vue:16]: Thank you! This has been fixed
Screen Shot 2020-01-31 at 6 05 53 PM

File: src/components/NewUserForm.vue [hunk 2]
```diff
@@ -29,4 +75,7 @@
         width: 70%;
         margin: 0 auto
     }
+    div.control.input-field.select::after {
+        display: none;
+    }
```
Comment (sschipke) [inline src/components/NewUserForm.vue:80]: This is to remove the extra arrow from the select input. It was being added by the component library.
Screen Shot 2020-01-31 at 6 05 53 PM

File: src/components/NewUserForm.vue (hunk 1 above)
Comment (nancylee713) [reply src/components/NewUserForm.vue:16]: Nice!

**Pull request merged**